### DISCUSSION TITLE
用docker,IPV6总是不能取得网卡接口地址

### DISCUSSION THREAD

**Discussion by wowhomg** [jeessy2/ddns-go#720] [Q&A]
重新刷机后第一次安装DDNS-go,是正常可以获取得到IPV6网卡的，也能正常外网访问，但一重启软路由，IPV6就会显示，未找到可用的网卡，IPV4就是正常的，然后改用“通过接口获取”总是返回错误说：连接失败! <a target='blank' href='https://speed.neu6.edu.cn/getIP.php'>点击查看接口能否返回IPv6地址</a>, 参考说明:<a target='blank' href='https://github.com/jeessy2/ddns-go#使用ipv6'>点击访问</a>
2023/06/03 13:49:52 错误信息: Get "https://speed.neu6.edu.cn/getIP.php": context deadline exceeded (Client.Timeout exceeded while awaiting headers)，但我试着打开这个地址，它又是正常的反回IPV6地址的，然后通过命令获取，用“仅获取纯公网 IPv6 地址”命令时会反回错误：失败！Code: AuthFailure.SecretIdNotFound, Message: The SecretId is not found, please ensure that your SecretId is correct.。用ip -6 addr show eth1“命令时反回错误”获取IPv6结果失败! 未能成功执行命令：/bin/bash -rc ip -6 addr show eth1，错误："ip: can't find device 'eth1'\n"，退出状态码：exit status 1
2023/06/03 13:53:40 未能获取IPv6地址, 将不会更新“，试过两款不同的软路由都是这样，但，在PC上它又是正常的。

**Comment by WaterLemons2k** [comment]
看最后通过命令获取的报错是找不到 `eth1`，运行：

```shell
ip link show
```

看下系统里有哪些网卡。

参考：https://www.cyberciti.biz/faq/linux-list-network-interfaces-names-command/

**Reply by Jengro777** [reply]
我也有相同的问题，使用命令获取ipv6 有出现 新的问题 #786
